test/functional/feature_bip8.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+# Test BIP 8 softforks
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+class Bip8Test(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 3
+        self.setup_clean_chain = True
+        # Node 0 has TestDummy inactive
+        # Node 1 has a regular activation window
+        # Node 2 uses speedy trial.
+        self.extra_args = [
+            [f'-vbparams=testdummy:@1:@1'],
+            [f'-vbparams=testdummy:@144:@{144 * 3}'],
+            [f'-vbparams=testdummy:@144:@{144 * 2}:@{144 * 4}'],
```

Comment:
(and below) f''should be avoided to make backport to 0.21 nicer

Reply:
Done